app/ReadmeFormatter.php

```diff
@@ -24,6 +24,8 @@ public function __construct($package)
 
     public function format($text)
     {
+        $text = $this->removeAnchors($text);
```

Comment:
Why are we removing these? Are they not useful? Thanks!

Reply:
![image](https://user-images.githubusercontent.com/1141514/94215806-88ade700-fe92-11ea-9711-0810263b354b.png)

My memory is a little fuzzy but I realized that anchors within an html readme link to a broken link.

The link highlighted in the screenshot above is
`https://github.com/timothyasp/nova-badge-field/blob/master/#laravel-nova-badge-field` and `blog/master/` should not be included in it.

Markdown formatted readmes do not have anchors for titles so I figured stripping them would be a good move. Alternatively we could probably add anchors to markdown readmes but I don't think many people would utilize that feature.

Reply:
OK, sounds good. I think we can consider re-adding them later in a way that works, but good for me for now.